stdout of test 'tst607` in directory 'src/mal` itself:


# 13:31:54 >  
# 13:31:54 >  Mtimeout -timeout 60 Mserver "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --debug=10 --set "monet_mod_path=/ufs/mk/monet5/Linux/lib/MonetDB5:/ufs/mk/opensource/MonetDB/Linux/lib/MonetDB" --set "gdk_dbfarm=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/log" --set mapi_port=32003 --set sql_port=41971 --set xquery_port=59185 --set monet_prompt= --trace "--config=/ufs/mk/monet5/src/mal/Tests/All.conf" --dbname=mTests_src_mal  tst607.mal < /dev/null
# 13:31:54 >  

# Monet Database Server V4.99.19
# Copyright (c) 1993-2005, CWI. All rights reserved.
# Compiled for x86_64-redhat-linux-gnu/64bit with 32bit OIDs; dynamically linked.
# config:/ufs/mk/monet5/src/mal/Tests/All.conf
# dbfarm:/ufs/mk/opensource/MonetDB/Linux/var/MonetDB5/dbfarm
# dbname:mTests_src_mal
# Visit http://monetdb.cwi.nl/ for further information.
## using spaces
#
#u:str:=const.take("user");
#io.printf("User bound to %s\n",u);
#roomnumber:= 321;
#const.deposit("roomnumber",roomnumber);
#
#v:int:= const.take("roomnumber");
#io.print(v);
#v:= 123;
#v:= v+1;
#const.release("roomnumber");
#io.print("attempt twice to release");
#const.release("roomnumber"); # failure? done twice or implicit take?
#
function user.main():void;	# 0  (main:void)
# using spaces 	# 1  (_1:str)
    u:str  := const.take("user");	# 2 CSTtake (u:str)<-(_3:str)
    io.printf("#User bound to %s\n",u);	# 3 IOprint_formatted_str (_4:int)<-(_5:str)(u:str)
    roomnumber := 321;	# 4  (roomnumber:int)<-(_7:int)
    const.deposit("roomnumber",roomnumber);	# 5 CSTdeposit (_8:void)<-(_9:str)(roomnumber:int)
    v:int  := const.take("roomnumber");	# 6 CSTtake (v:int)<-(_9:str)
    io.print(v);	# 7 IOprint_val (_11:int)<-(v:int)
    v:int  := 123;	# 8  (v:int)<-(_12:int)
    v:int  := calc.+(v,1);	# 9 CALCbinaryADDintint (v:int)<-(v:int)(_13:int)
    const.release("roomnumber");	# 10 CSTrelease (_14:void)<-(_9:str)
    io.print("attempt twice to release");	# 11 IOprint_val (_15:int)<-(_16:str)
    const.release("roomnumber");	# 12 CSTrelease (_17:void)<-(_9:str)
# failure? done twice or implicit take? 	# 13  (_18:str)
end main;	# 14  
#User bound to mk
[ 321 ]
[ "attempt twice to release" ]

# 13:31:54 >  
# 13:31:54 >  Done.
# 13:31:54 >  

